frame/ddc-metrics-offchain-worker/src/lib.rs

```diff
@@ -102,6 +102,20 @@ impl MetricInfo {
     }
 }
 
+pub fn get_contract_id() -> [u8; 32] {
+	let sc_address_store = StorageValueRef::persistent(b"ddc-metrics-offchain-worker::sc_address");
```

Comment:
Are we able to set this value from the UI or CLI?

Reply:
Yes, sure

Reply:
What is the format in the block viewer?

Contract address must decode to [u8; 32] (fixed-size array)
DDC URL must decode to Vec<u8> (UTF-8 string as bytes)

Decoding using [CODEC](https://substrate.dev/docs/en/knowledgebase/advanced/codec#vectors-lists-series-sets).

Reply:
Let me check

Reply:
Seems like block viewer sends it as byte array

Reply:
@naure 
Will merge it as is. At least we can configure SC address.